modules/nf-core/glimpse/ligate/main.nf

```diff
@@ -0,0 +1,35 @@
+process GLIMPSE_LIGATE {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::glimpse-bio=1.1.1"
```

Comment:
Latest version on github is 2.0.0. Could you look into updating bioconda and then your module (in the future)

Reply:
Hi!
I'm currently working on that [PR](https://github.com/bioconda/bioconda-recipes/pull/38994)
But there is some problem for the moment, and the two version are not exactly for the same use.

Reply:
Way ahead of me 👍🏻